lib/change_health/response/claim/report/report_277_data.rb

```diff
@@ -0,0 +1,76 @@
+module ChangeHealth
+  module Response
+    module Claim
+      # Methods only return meaningful data for json reports
+      class Report277Data < ChangeHealth::Response::Claim::ReportData
+        def transactions
+          @raw.dig('transactions')
+        end
+
+        # Only one payer per report
+        def payer_org_name
+          transactions&.first&.dig('payers')&.first&.dig('organizationName')
+        end
+
+        def claims
+          report_claims = []
+
+          transactions&.each do |transaction|
```

Comment:
All this dig & each is so ugly but I don't know a better way considering the json that it's parsing is also pretty gnarly. See example json in test/samples below. Suggestions welcome!

Reply:
To make it prettier the only thing I can think is make a subclass that handles it sort of like the https://github.com/WeInfuse/change_health/blob/master/lib/change_health/response/eligibility/eligibility_data.rb class?

Also, rubocop would say not to use `dig` if you're only going 1-level deep and instead access with [] (I think just because it's shorter?).

Use `dig` for 2+ levels like when you do `patient_claim_status_detail.dig('subscriber')&.dig('firstName')` that is actually equivalent to `patient_claim_status_detail.dig('subscriber', 'firstName')`